axis.go

```diff
@@ -490,14 +490,14 @@ type DefaultTicks struct{}
 var _ Ticker = DefaultTicks{}
 
 // Ticks returns Ticks in the specified range.
-func (DefaultTicks) Ticks(min, max float64) []Tick {
-	if max <= min {
+func (DefaultTicks) Ticks(minTick, maxTick float64) []Tick {
```

Comment:
> In order to use the builtin, we need to reduce the simplicity of the documentation by changing the return parameter names.

I assume you are referring to the change here? I’d love to understand your perspective better because, from my view, this change doesn’t reduce simplicity, it only modifies the parameter names, which are internal details and do not impact the API or how users interact with the function. `DefaultTicks` still fulfils the `Ticker` interface.

If there's a particular case where you think it might cause confusion, I'd be happy to explore alternatives.

Reply:
> I assume you are referring to the change here?

Yes.

> I’d love to understand your perspective better because, from my view, this change doesn’t reduce simplicity, it only modifies the parameter names,

This is the issue. It changes the names of the returned values from the consistently used names. This is a reduction in simplicity.

*which are internal details* and do not impact the API or how users interact with the function.

This is definitively not true. These names are published in the public godoc and so impact on the client user's interaction with the API via that surface.

This change is not necessary. Let's drop it please.

Reply:
Thanks for the feedback. I have reverted the change in this file in commit e0aeedfeb94ca850e3aefdc62dbeff82fa87b113.